guides/user/canary/config/index.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: single
+title:  "Configure a canary"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Before you can add a canary stage to a pipeline, you need to configure what the
+canary consists of, including...
+
+* The specific metrics to evaluate, and a logical grouping of those metrics
+* Default scoring thresholds (which can be overridden in the
+canary stage)
+* Optionally, one or more filter templates
+* A name by which a canary stage can choose this config
```

Comment:
Not a big deal, but the first three follow the UI's top-bottom order, but the last bullet does not.